src/main/java/seedu/address/storage/JsonAdaptedGameEntry.java

```diff
@@ -108,11 +113,17 @@ public GameEntry toModelType() throws IllegalValueException, ParseException {
         }
         // todo: add input validation check for date
         DatePlayed modelDate = new DatePlayed();
+        String datePlayedString = null;
         try {
-            modelDate = new DatePlayed(new SimpleDateFormat("yyyy-MM-dd hh:mm").parse(date));
+            datePlayedString = DATETIME_INPUT_FORMAT.format(DATETIME_FORMAT.parse(date));
```

Comment:
I thought we're just supposed to check the string validity using DatePlayed.isValidDatePlayed()?

Reply:
I realized that we need to do this because the format that we save dates in Storage is different from how we input them in command.  We should clean this up in the future, but for now I don't think that we should move the checking of storage into DatePlayed since it violates encapsulation.